sphinxext/opengraph/descriptionparser.py

```diff
@@ -124,5 +124,4 @@ def get_description(
 
     mcv = DescriptionParser(description_length, known_titles, document)
     doctree.walkabout(mcv)
-    # Parse quotation so they won't break html tags if smart quotes are disabled
-    return mcv.description.replace('"', "&quot;")
+    return mcv.description
```

Comment:
This should be in the separate PR from above.

Reply:
again, since this already was in a separate PR, it just gets moved so that it will also run on tags added via field lists, no real point in separating it since up until this PR the old location worked identically.